### DISCUSSION TITLE
Windows Defender also detects trojan in the .jar file

### DISCUSSION THREAD

**Discussion by Glisterine** [jagrosh/MusicBot#989] [Questions]
I've been using the .jar for like Two Weeks, and today when I wanted to use it, Windows Defender immediately detects Trojan in it, and block it.
![Errors](https://user-images.githubusercontent.com/92537466/137355494-d2ce9269-8180-4bb7-a81e-aac5901fa087.png)

Is it a False Positive or else?

**Comment by jkiller295** [comment]
False Positive according to the answers I got here
https://github.com/jagrosh/MusicBot/discussions/958
